src/algebra/invertible.lean

```diff
@@ -155,6 +155,17 @@ def invertible.copy [monoid α] {r : α} (hr : invertible r) (s : α) (hs : s =
   inv_of_mul_self := by rw [hs, inv_of_mul_self],
   mul_inv_of_self := by rw [hs, mul_inv_of_self] }
 
+theorem commute.inv_right_invertible [monoid α] {a b : α} [invertible b] (h : commute a b) :
+  commute a (⅟b) :=
+calc a * (⅟b) = (⅟b) * (b * a * (⅟b)) : by simp [mul_assoc]
+... = (⅟b) * (a * b * ((⅟b))) : by rw h.eq
+... = (⅟b) * a : by simp [mul_assoc]
+
+theorem commute.inv_left_invertible [monoid α] {a b : α} [invertible b] (h : commute b a) :
```

Comment:
```suggestion
theorem commute.inv_of_left [monoid α] {a b : α} [invertible b] (h : commute b a) :
```

Reply:
I wanted to keep the name similar to `commute.inv_left` and `commute.inv_left'`, which deal respectively with `⁻¹` (in a group) and `⁻¹` (in a `group_with_zero`).  Then there's also `commute.units_inv_left`, dealing with `⁻¹` for a unit in a `monoid` ...

Reply:
Well, `⁻¹` is called `inv` and `⅟` is called `inv_of`, so surely my suggestion meets that requirement?

Reply:
As an example: `inv_one` is the group lemma, `inv_of_one` is the `invertible` lemma.

Reply:
Ah, I didn't know!  Sounds good then.